umrr_ros2_driver/CMakeLists.txt

```diff
@@ -30,8 +30,11 @@ endif()
 
 find_package(ament_cmake_auto REQUIRED)
 find_package(Threads REQUIRED)
+find_package(umrr_ros2_msgs REQUIRED)
```

Comment:
This isn't necessary if you add this package to `package.xml` and then call `ament_auto_find_build_dependencies()` (which you should, because you are using `ament_cmake_auto`).